cardano-cli/src/blockchain/commands.rs

```diff
@@ -343,7 +344,11 @@ pub fn cat( mut term: Term
         use utils::pretty::Pretty;
 
         let blk = rblk.decode().unwrap();
-        blk.pretty(&mut term, 0).unwrap();
+        if debug {
+            println!("{:?}", blk);
```

Comment:
use `{:#?}` instead of just `{:?}`. It will pretty print the rust structure over multilines and all. Easier to understand overall.

Reply:
and don't `println` use `writeln!(term, "{:#?}", blk)`. So we still have control on the output redirection. This will become handy when we need to _mock_ the output behavior into test.